src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/CountOperation.java

```diff
@@ -1,6 +1,7 @@
 package io.stargate.sgv2.jsonapi.service.operation.model;
 
 import com.bpodgursky.jbool_expressions.Expression;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
 import io.smallrye.mutiny.Uni;
 import io.stargate.bridge.proto.QueryOuterClass;
```

Comment:
can this reference to QueryOuterClass be removed? (and this is probably a global question)

Reply:
Or maybe full removal of bridge related stuff would be an additional cleanup step we do later?

Reply:
We still use the QueryBuilder from the stargate api commons to generated queries with nested OR conditions. Driver based query builder still don't have this or atleast not documented anywhere.

Reply:
Ok for now. Let's make sure there is a follow up issue to track remaining bridge removal